epi/src/lib.rs

```diff
@@ -155,6 +155,12 @@ pub trait App {
     fn transparent(&self) -> bool {
         false
     }
+
+    /// On Windows: enable drag and drop support.
+    /// Set to false to avoid issues with crates such as cpal which uses that use multi-threaded COM API <https://github.com/rust-windowing/winit/pull/1524>
+    fn drag_and_drop_support(&self) -> bool {
```

Comment:
perhaps it is better to keep this as a egui_glium-specific setting instead somehow? add arguments to eframe::run_native?

Reply:
Yeah, I think this and a few of other things in `trait App` (`is_resizable`, `icon_data`, `decorated`, and `transparent`) should be moved to some options that is passed to `eframe::run_native`, but we can do that in a separate PR